src/utils/batteryParameters.js

```diff
@@ -0,0 +1,99 @@
+//en: English language code (Global)
+//de: German language code (Germany)
+//ar: Arabic language code (UAE)
+//ms: Malay language code (Singapore)
+
+const parameters = [
```

Comment:
Well thought-out structuring in the `parameters` here :+1: 
Think of a method to further mistake-proof it.
For instance: Think of adding Spanish. A developer may miss adding the message for one of the conditions. We will not know till the actual program is run and the issue is noticed manually.
Think of a test that can catch this issue. Try to make a test without having to exercise the whole program for every single language.
Hint: Think of validating this structure (via Json schema?) or the function that converts the condition to a message